framework/src/base/ComputeResidualThread.C

```diff
@@ -101,7 +80,7 @@ ComputeResidualThread::onElement(const Elem *elem)
   _fe_problem.reinitMaterials(_subdomain, _tid);
 
 
-  const MooseObjectWarehouse<KernelBase> * storage;
+  MooseObjectWarehouse<KernelBase> const * storage;
```

Comment:
I'm not sure if this change was intentional or not, but it is fairly confusing. Usually when we see const **not** at the front of the declaration, we take that to mean that the pointer is constant, not the data (e.g. `Foo * const p`). However, in this case the const actually appears in-between the type and the pointer designation which means const data which is equivalent to the more normal declaration `const Foo * p`. So this change is equivalent and unecessary and just plain confusing.